docs/agent_hocon_reference.md

```diff
@@ -350,6 +351,29 @@ This is an optional list of string keys in the [properties](#properties) diction
 to be required whenever an upstream agent calls the one being described.
 Note that it's possible to specify a default value for any property that is not listed as required.
 
+#### sly_data
+
+The optional [JSON Schema](https://json-schema.org) dictionary describing what
+specific information the agent needs as input arguments over the private sly_data dictionary
+channel when it is called.  The sly_data itself is generally considered to be private information
+that does not belong in the chat stream, i.e. credential information.
+
```

Comment:
"for example" credential information?
Because right now it reads sly_data == credentials

Reply:
Good suggestion. Adopted.